data/json/items/resources/wood.json

```diff
@@ -98,28 +98,64 @@
     "name": { "str": "plank" },
     "description": "A narrow, thick plank of wood, like a 2 by 4 or similar piece of dimensional lumber.  Makes a decent melee weapon, and can be used for all kinds construction.",
     "category": "spare_parts",
-    "weight": "2600 g",
+    "weight": "2440 g",
     "color": "brown",
     "symbol": "/",
     "material": [ "wood" ],
     "techniques": [ "WBLOCK_1" ],
-    "volume": "4400 ml",
-    "longest_side": "130 cm",
+    "volume": "4130 ml",
+    "longest_side": "122 cm",
     "bashing": 26,
     "to_hit": { "grip": "none", "length": "long", "surface": "any", "balance": "uneven" },
     "price": 1000,
     "price_postapoc": 10,
     "flags": [ "FRAGILE_MELEE", "FIREWOOD" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "plank_short",
+    "name": { "str": "short plank" },
+    "description": "A short 2 by 4 plank of wood.  While less common and smaller than its 4 foot cousin it can still be very useful in carpentry.",
+    "category": "spare_parts",
+    "weight": "1220 g",
+    "color": "brown",
+    "symbol": "/",
+    "material": [ "wood" ],
+    "volume": "2066 ml",
+    "longest_side": "61 cm",
+    "bashing": 6,
+    "to_hit": { "grip": "none", "length": "long", "surface": "any", "balance": "uneven" },
+    "price": 500,
+    "price_postapoc": 10,
+    "flags": [ "FRAGILE_MELEE", "FIREWOOD" ]
```

Comment:
> I also gave it "Fragile_melee" as while it is sturdy in a vacuum, the planks are long enough that repeatedly hitting people with it would eventually snap it in half.

> #56953 / where the regular plank was given FRAGILE_MELEE

I don't think this applies if it's only a two footer

Reply:
Technically you are right and, to be honest, I've never tried hitting people with a 2 foot plank to check how sturdy it is. It kind of feels wrong to have it _not_ be fragile since its even smaller and weaker than the regular 4 foot plank but I do understand where you're coming from with this.